frontend/src/pages/Home.js

```diff
@@ -0,0 +1,33 @@
+// Components
```

Comment:
This page shouldn't exist anymore. You have `HomePage` now. Probably leftover from rebase.

Reply:
I don't know how this happened :) 
So how do I proceed to remove this? Rebase again?

Reply:
I'm guessing you may've chosen the "wrong thing to keep" when solving conflicts.

Reply:
I think rebasing again won't work.

You can delete the pages and do `git commit --amend --no-edit` which will add those changes to your last commit. Then, I'd make sure to squash your 2 commits into one. But Git master @DavideSilva may have another suggestion.

Reply:
I think it's done now :)

Reply:
No, that's usually what I do when I mess up and end up with changes that I don't want 👍